packages/core/app-layout/src/components/sidebar/AppSidebar.vue

```diff
@@ -68,37 +68,41 @@
         </nav>
       </div>
 
-      <SidebarFooter
-        v-if="profileName || profileItems.length"
-        class="sidebar-profile-menu"
-        :item-count="profileItems.length"
-        :name="profileName"
-      >
-        <KDropdownItem
-          v-for="item in profileItems"
-          :key="item.name"
-          :class="[{ 'has-divider': item.hasDivider },{ 'external-profile-dropdown-link': (item.external || item.newWindow) && typeof item.to === 'string' }]"
-          :has-divider="item.hasDivider"
-          :item="(item.external || item.newWindow) && typeof item.to === 'string' ? undefined : { label: item.name, to: item.to }"
-          @click="itemClick(item)"
-        >
-          <a
-            v-if="(item.external || item.newWindow) && typeof item.to === 'string'"
-            class="sidebar-item-external-link"
-            :href="item.to"
-            :target="item.newWindow ? '_blank' : undefined"
+      <div class="sidebar-footer">
+        <slot name="footer">
+          <SidebarFooter
+            v-if="profileName || profileItems.length"
```

Comment:
Should this condition be the negation? 
Since
>  If `footer` slot content is provided, it will override the `profileItems` and `profileName`, preventing the sidebar bottom profile menu and popup menu from appearing.

Reply:
No, this logic is unchanged. 

Since the `<SidebarFooter>` component is now the default slot content, it will only render if the host app _does not_ provide it's own `footer` slot content.